src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -230,6 +235,10 @@ public void StartConnectAsync()
                 return;
             }
 
+            // Connect if local, ip range filtering disabled or ip range filtering enabled and peer in a different group. 
+            if (!peerAddress.Endpoint.Address.IsLocal() && this.ConnectionSettings.IpRangeFiltering && this.PeerIsPartOfExistingGroup(peerAddress))
+                return;
```

Comment:
Fazz, before we return here might want to log this:

```
this.logger.LogTrace("(-) {0}:{1}", nameof(peerAddress.Endpoint), peerAddress.Endpoint, nameof(peerAddress.Endpoint.Address.IsLocal(), peerAddress.Endpoint.Address.IsLocal())...
```

@Aprogiena ?

Reply:
i'd go with just
```this.logger.LogTrace("(-)[RANGE_FILTERED]")```

Reply:
Done.